starter-code/index.html

```diff
@@ -1,31 +1,76 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <title>Instagram</title>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" media="screen" href="./css/main.css" />
+
 </head>
+
 <body>
-  Instagram
 
-  Capture and Share the World's Moments
+  <section>
+
+    <div id="phones">
+      <img src="images/phones.png">
+    </div>
+
+    <div id="content">
+
+      <header>
+        
+        <a href =""><img src="./images/brand.png" style="width: 80%"></a>
+        <button><img src="./images/home.png"><span id="home">Log in</span></button>
+      
+      </header>
+
+        <div id="capture">
+
+          <h1>Capture and Share the World's Moments</h1>
+
+          <p>Instagram is a <span>fast</span>, <span>beautiful</span> and <span>fun</span> way to share your life with friends and family.<br>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+          Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.<br>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+          Oh yeah, did we mention it's free?</p>
 
-  Oh yeah, did we mention it's free?
+          <div id="download">
+  
+            <img src="images/badge-iphone.png" style="margin-right:2%;">
+            <img src="images/badge-android.png" style="margin-left:2%;">
+  
+          </div>
 
-  Download on the App Store
-  Get it on Google Play
+        </div>
 
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
+    </div>    
+
+  </section>
+
+  <footer>
+
+  <nav>
+
+    <ul>
+      <li><a href="#">About Us</a></li>
+      <li><a href="#">Support</a></li>
+      <li><a href="#">Blog</a></li>
+      <li><a href="#">Press</a></li>
+      <li><a href="#">API</a></li>
+      <li><a href="#">Jobs</a></li>
+      <li><a href="#">Privacy</a></li>
+      <li><a href="#">Terms</a></li>
+    </ul>
+
+  </nav>
+
+    <p id="cw">&copy; 2014 Instagram</p>
+
```

Comment:
Well done guys! Few stuff to edit but the essential part of this lab (flex box, justify-content, align-items) are understood!

You rock 🖖